File: 0169-majority-element/0169-majority-element.c
int majorityElement(int* nums, int numsSize) {
    int c = 1;
    int n = numsSize;
    int ele=nums[0];
    for (int i=1;i<n;i++){
        if(nums[i]==ele){
            c++;
        }
        else{
            c--;
            if(c==0)
            {
                ele=nums[i+1];
            }
        }
    }
    return ele;
}

File: 0169-majority-element/0169-majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
        // int cnt=0;
        // int ele=nums[0];
        // int n=nums.length;
        Arrays.sort(nums);
        // for(int i=0;i<nums.length-1;i++){
        //     if(nums[i]==nums[i+1]){
        //         cnt++;
        //     }
        //     if(cnt>=Math.floor(n/2)){
        //         ele=nums[i];
        //         break;
        //     }
        // }
        // return ele;
            return nums[(nums.length/2)];

    }
}